apps/chat/README.md

```diff
@@ -9,7 +9,22 @@ The easiest way to get started with messaging in the Amazon Chime SDK is to depl
 - An [AWS account](https://aws.amazon.com/free/) that provides access to AWS services.
 - [Node.js](https://nodejs.org/en/download/) version 14 or higher.
 
-## Deploying the solution
+## Deploying the demo app
+### Using AWS SAM
+1. Run the following commands to provision the demo app's infrastructure, as well as build and deploy the web assets.
+
+```bash
+cd apps/chat
+npm run deploy
+```
+
+2. Verify that the `src/backend/serverless/appconfig.json` is populated with the CloudFormation stack outputs. This JSON file used to
+   configure the app with `src/Config.js.`
+
+3. Then, you can access the app via the CloudFront endpoint found in the CloudFormation stack outputs.
+
+### Manually
```

Comment:
nit: How about `Using the AWS CloudFormation console` (vs. `Using AWS SAM`) since running CLI commands also requires manual steps?